[2024-02-11T02:49:21.752+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: censoEnade.rename_censo_dir manual__2024-02-11T02:48:53.896374+00:00 [queued]>
[2024-02-11T02:49:21.765+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: censoEnade.rename_censo_dir manual__2024-02-11T02:48:53.896374+00:00 [queued]>
[2024-02-11T02:49:21.766+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-11T02:49:21.788+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): rename_censo_dir> on 2024-02-11 02:48:53.896374+00:00
[2024-02-11T02:49:21.796+0000] {standard_task_runner.py:60} INFO - Started process 329 to run task
[2024-02-11T02:49:21.800+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'censoEnade', 'rename_censo_dir', 'manual__2024-02-11T02:48:53.896374+00:00', '--job-id', '561', '--raw', '--subdir', 'DAGS_FOLDER/censoEnade.py', '--cfg-path', '/tmp/tmpmisg9d2q']
[2024-02-11T02:49:21.802+0000] {standard_task_runner.py:88} INFO - Job 561: Subtask rename_censo_dir
[2024-02-11T02:49:21.865+0000] {task_command.py:423} INFO - Running <TaskInstance: censoEnade.rename_censo_dir manual__2024-02-11T02:48:53.896374+00:00 [running]> on host 7e5b5e202339
[2024-02-11T02:49:21.984+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Néfi Fernandes' AIRFLOW_CTX_DAG_ID='censoEnade' AIRFLOW_CTX_TASK_ID='rename_censo_dir' AIRFLOW_CTX_EXECUTION_DATE='2024-02-11T02:48:53.896374+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-11T02:48:53.896374+00:00'
[2024-02-11T02:49:21.989+0000] {logging_mixin.py:188} INFO - O diretório censo_ed_superior_2021 já existe. Continuando para a próxima etapa.
[2024-02-11T02:49:21.990+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-11T02:49:22.002+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=censoEnade, task_id=rename_censo_dir, execution_date=20240211T024853, start_date=20240211T024921, end_date=20240211T024922
[2024-02-11T02:49:22.053+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-11T02:49:22.082+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
